caffe2/python/onnx/tests/onnx_backend_test.py

```diff
@@ -13,6 +13,9 @@
 
 import caffe2.python.onnx.backend as c2
 
+from caffe2.python import core
+core.SetEnginePref({}, {})
```

Comment:
What's the purpose of this code?

Reply:
So that we test the default gpu implementation (instead of cudnn and miopen). Ideally we probably should test both (use cudnn and not).